### PR TITLE
Pd 541 fleet dashboard article

### PR DESCRIPTION
**Pull request by micjohnson777**
This PR adds content to the new Fleet Dashboard article, creates a snippet for the system options menu, and updates the main Dashboard article content.

A future PR will add links to the system option menu snippet that open other articles on the function after those articles are updated and reorganized.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/TrueCommand/UserGuide/FleetDashboard.md [hunk 1]
```diff
@@ -2,6 +2,72 @@
...
 weight: 15
+tags:
+- tcsystem
+- tcdashboard
+- tcalerts
+- tcupdate
 ---
 
-Placeholder
+In addition to the main TrueCommand [**Dashboard**]({{< relref "/TrueCommand/UserGuide/Dashboard.md" >}}) that shows individual system cards with performance  information for each system managed by TrueCcommand, release 3.0 introduces a new **Fleet Dashboard** with new function widgets (system cards).
```
Comment (DjP-iX) [inline content/TrueCommand/UserGuide/FleetDashboard.md]: "function widgets (system cards)" Should be consisistent with line 16

File: content/TrueCommand/UserGuide/FleetDashboard.md [hunk 2]
```diff
@@ -2,6 +2,72 @@
...
+
+![FleetDashboard](/images/TrueCommand/Dashboard/FleetDashboard.png "Fleet Dashboard")
+
+The **Fleet Dashboard** provides an overview of system information organzed into six widgets (information card):
```
Comment (DjP-iX) [inline content/TrueCommand/UserGuide/FleetDashboard.md]: "widgets (information cards)" Should be consistent with line 12

File: content/TrueCommand/UserGuide/FleetDashboard.md [hunk 3]
```diff
@@ -2,6 +2,72 @@
...
+
+To view more information on shares, replication, pools, and alerts, go to the setting menu option an open the related system or alert screen.
+
+## Systems Widget
+
+The **Systems** widget lists all systems added to and managed by TrueCommand. 
+
+![FleetDashboardSystemsWidget](/images/TrueCommand/Dashboard/FleetDashboardSystemsWidget.png "Fleet Dashboard Systems Widget")
+
+The card includes the **Systems** button that opens the [**New System**]() screen, and the [system options menu](#system-options-menu) that provides access to other functions available to each system managed by the TrueCommand interface.
```
Comment (DjP-iX) [inline content/TrueCommand/UserGuide/FleetDashboard.md]: `"[**New System**]() screen"` is missing a link

File: content/TrueCommand/UserGuide/FleetDashboard.md [hunk 4]
```diff
@@ -2,6 +2,72 @@
...
+
+To access system details and management functions, click the gear gear <i class="material-icons" aria-hidden="true" title="Settings">settings</i> icon, then select either **System Inventory**, or if an administration user, **Systems** to open those screens.
```
Comment (DjP-iX) [inline content/TrueCommand/UserGuide/FleetDashboard.md]: Is this accurate? I'm not seeing a gear in the screenshot?

**Review by DjP-iX — CHANGES_REQUESTED** [review]

File: content/TrueCommand/UserGuide/FleetDashboard.md (hunk 3 above)
Comment (micjohnson777) [reply content/TrueCommand/UserGuide/FleetDashboard.md]: Deleted this link until I can create an article or add this content to an existing article.

File: content/TrueCommand/UserGuide/FleetDashboard.md (hunk 4 above)
Comment (micjohnson777) [reply content/TrueCommand/UserGuide/FleetDashboard.md]: I replaced the icon with one that works.

**Review by DjP-iX — APPROVED** [review]

**Pull request merged**